packages/block-library/src/post-title/style.scss

```diff
@@ -1,3 +1,7 @@
 .wp-block-post-title a {
 	display: inline-block;
+
+	// Inherit the styles from the main block element, so that text decoration settings
+	// for the block apply to post title links.
+	text-decoration: inherit;
```

Comment:
This is basically a hack to apply the inline text-decoration styles to the child `<a>` element, when the post title uses a link. One undesired side-effect is that the default underlined browser link style is overridden (an accessibility concern).

Alternatives I can think of

- Update `edit.js` and `render_block_core_post_title` for this block to apply the inline styles to `<a>`, when present (rather than the parent heading element)
- Update the experimental style support to allow targeting an element in the block, (e.g. [how duotone works](https://github.com/WordPress/gutenberg/blob/b9232527214cd3d523badc4a66b529b7efe615ca/docs/reference-guides/block-api/block-supports.md#color__experimentalduotone))

Reply:
cc @nosolosw , as I believe the ability to specify styles that target elements within blocks is under active development.

Reply:
At the block level (hooks), I think we'd want to use the `__experimentalSkipSerialization` flag. The post title block would be responsible for serializing the value directly to the `a` element. Context: https://github.com/WordPress/gutenberg/issues/28913 and this particular PR is necessary for this to work https://github.com/WordPress/gutenberg/pull/30880 

At the global level (via theme.json), we can already target `a` elements within the post title block:

```
styles: {
    core/post-title: {
        elements: {
            link: {
                typography: {
                    textDecoration: "value"
                }
            }
        }
    }
}
```

Reply:
Since https://github.com/WordPress/gutenberg/pull/30880 has landed, should this PR be refreshed?